jenkins/kola/dev-container.sh

```diff
@@ -42,8 +42,8 @@ emerge-gitclone
 . /usr/share/coreos/release
 if [[ $FLATCAR_RELEASE_VERSION =~ master ]]
 then
-        git -C /var/lib/portage/portage-stable checkout master
-        git -C /var/lib/portage/coreos-overlay checkout master
+        git -C /var/lib/portage/portage-stable checkout main
```

Comment:
I think that the recent change in `emerge-gitclone` would have been enough to ensure that the right branch gets checked out, or? Seems like we can delete this or we should actually stop checking out `main` here and rather use the right branch that is specified in the manifest.

Reply:
Deleted the lines